packages/ometrics/ometrics.0.1.2/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+license: "MIT"
+maintainer: "Valentin Chaboche <valentin.chaboche@lambda-coins.com>"
+homepage: "https://gitlab.com/nomadic-labs/ometrics"
+dev-repo: "git+https://gitlab.com/nomadic-labs/ometrics.git"
+bug-reports: "https://gitlab.com/nomadic-labs/ometrics/-/issues"
+synopsis: "OCaml analysis in a merge request changes"
+
+depends: [
+  "ocaml" {>= "4.12" & < "4.13"}
```

Comment:
```suggestion
  "ocaml" {>= "4.12"}
```
What is the reason for such a tight constraint?

Reply:
The internal of this lib is based on the OCaml AST, if I remember correctly it does not compile with 4.13, I'll check

Reply:
Ah I see it uses a vendored merlin. I tested. It works with 4.13 but not 4.14